From 4730a71eedac40d2b406ca8720a3e0c75fc26543 Mon Sep 17 00:00:00 2001
From: David Disseldorp <ddiss@suse.de>
Date: Wed, 2 Jan 2019 14:46:04 +0100
Subject: [PATCH] target: se_dev_attrib.emulate_pr ABI stability
References: bsc#1091405
Patch-mainline: Never, ABI only

struct se_dev_attrib has a new emulate_pr member, which consumes the
position of the unused and now removed pi_prot_format member.

Signed-off-by: David Disseldorp <ddiss@suse.de>
---
 include/target/target_core_base.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/target/target_core_base.h b/include/target/target_core_base.h
index b05af8899eec..aaf0028318c2 100644
--- a/include/target/target_core_base.h
+++ b/include/target/target_core_base.h
@@ -683,7 +683,11 @@ struct se_dev_attrib {
 	int		emulate_tpws;
 	int		emulate_caw;
 	int		emulate_3pc;
+#ifdef __GENKSYMS__
+	int		pi_prot_format;
+#else
 	int		emulate_pr;
+#endif
 	enum target_prot_type pi_prot_type;
 	enum target_prot_type hw_pi_prot_type;
 	int		enforce_pr_isids;
-- 
2.13.7

